From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: OpenWrt bring-up <openwrt-bringup@example.invalid>
Date: Mon, 10 Aug 2026 00:00:00 +0800
Subject: [PATCH] ARM: dts: add ZTE ZX279128S E2631

Register the ZTE vendor DTS directory for first-stage OpenWrt initramfs
bring-up.

Signed-off-by: OpenWrt bring-up <openwrt-bringup@example.invalid>
---
 arch/arm/boot/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -105,3 +105,4 @@ subdir-y += ti
 subdir-y += uniphier
 subdir-y += versatile
 subdir-y += xilinx
+subdir-y += zte
